app/templates/mock-login.hbs

```diff
@@ -1,54 +1,37 @@
-<div id="main" itemprop="mainContentOfPage" role="main" tabindex="-1" class="au-c-main-container__content au-c-main-container__content--scroll">
+<AuBodyContainer @scroll={{true}} itemprop="mainContentOfPage">
+  <main id="content">
```

Comment:
It seems our application template already contains a main element, so I would switch this to a regular `div` instead.

![image](https://user-images.githubusercontent.com/3533236/195345705-6cfb25cf-3d6d-4542-8e02-d45503d2e57f.png)